python/tests/graph/test_utils.py

```diff
@@ -42,7 +42,31 @@ def _gen_invalid_tensor_or_op_input_with_wrong_types():
         yield TestCase(data=wrong_val, description='wrong type {}'.format(type(wrong_val)))
 
 
-def _gen_valid_tensor_op_objects():
+def _gen_invalid_tensor_or_op_with_graph_pairing():
+    tnsr = tf.constant(1427.08, name='someConstOp')
+    other_graph = tf.Graph()
+    op_name = tnsr.op.name
+
+    # Test get_tensor and get_op returns tensor or op contained in the same graph
+    yield TestCase(data=lambda: tfx.get_op(tnsr, other_graph),
+                   description='test graph from getting op fron tensor')
```

Comment:
'test get_op from tensor with wrong graph' is more clear (similarly for other testcases in this method)